### PR TITLE
Add Examples to bottom of every function page.

### PR DESCRIPTION
**Pull request by etefera**
Fixes https://github.com/GoogleContainerTools/kpt/issues/2268
image
image

### CODE REVIEW

File: site/index.html [hunk 1]
```diff
@@ -128,7 +134,43 @@
...
+                  exampleNames.forEach((ex, _) => {
+                    if (currentEx != ex) {
+                      exampleItems += `<li><a href="${examples[ex].LocalExamplePath}/">
+                      ${ex}</a></li>`;
+                    }
+                  });
+                  let examplesInlineListSection = exampleItems
+                    ? `
+                  <h3 id="examples">${
+                    currentEx ? "Other Examples" : "Examples"
```
Comment (frankfarzan) [inline site/index.html:167]: "More Examples" might be better

**Comment by droot** [comment]
Not scoped to this PR, but I realized, we need to come up with better labels for these examples. For ex: `suffix` --> `Add suffix in resource names` something more descriptive. @mengqiy

File: site/index.html (hunk 1 above)
Comment (etefera) [reply site/index.html]: Done

**Comment by mengqiy** [comment]
> we need to come up with better labels for these examples. For ex: suffix --> Add suffix in resource names something more descriptive.

There are 2 things here:
- The name of the example that will show up in the site (we want it to be descriptive)
- Package path that can be fetched using `kpt pkg get` (we want it to be short)

Eyob and I had some discussion recently about changing the `metadata.yaml` file. Currently we have `examples` which is a string list. We can change it to be a map from example name on site to the example URL. Then we can make name descriptive while the URL short.

**Review by mengqiy — APPROVED** [review]

**Pull request merged**